docs/UserGuide.md

```diff
@@ -164,23 +166,61 @@ A member of our core team will attend to you as soon as possible.
 
 ## User Interface
 
-{to be filled in}
+`EduMate`'s UI components are built using FXML and JavaFX. Each of the UI components are built to fulfil a specific functionality. We also used light purple and white as our main colour scheme as they are cool colours.
+
+![Ui](images/UG_ui_pictures/user_interface.png)
+<center>Diagram 2: <code>EduMate</code>'s UI</center>
 
 ### Person List
 
-{to be filled in}
+You may locate the Person List at the center-left of `EduMate`. This is where you will see all your saved contacts (or sample data if you are trying out for the first time).
+
+<img src="images/UG_ui_pictures/contact_list.png" style="width:80%;margin:0 10%">
+
+<center>Diagram 3: Person List with some contacts displayed.</center>
 
 ### Profile Panel
 
-{to be filled in}
+The Profile Panel allows you to display your own or your contact's information.
+
+The Profile Panel will display basic information like your / contact's name, phone number, email address, nearest home station and the modules you are taking in NUS.
```

Comment:
Nearest home station sounds a little strange to me, what about: 
"The Profile Panel will display basic information, including name, phone number, email address, nearest MRT station based on the respective address, and NUS modules taken, for either you or your contacts."